File: internal/types/log.go
package types

// SimpleLog 简洁日志条目
type SimpleLog struct {
	Date    string `json:"date"`    // 日期，格式：2006/1/2
	Time    string `json:"time"`    // 时间，格式：15:04:05
	Message string `json:"message"` // 日志内容
}

// LogQuery 日志查询参数
type LogQuery struct {
	Keyword string `json:"keyword"` // 关键词搜索
	Limit   int    `json:"limit"`   // 返回条数，默认100
}

File: frontend/src/types/log.ts
// 日志类型定义

// 简洁日志条目
export interface SimpleLog {
  date: string    // 日期，格式：2006/1/2
  time: string    // 时间，格式：15:04:05
  message: string // 日志内容
}

// 日志查询参数
export interface LogQuery {
  keyword?: string  // 关键词搜索
  limit?: number    // 返回条数，默认100
}
